sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -733,6 +724,33 @@ struct _pi_context : _pi_object {
   std::unordered_map<void *, MemAllocRecord> MemAllocs;
 
 private:
+  // Calculate root device.
```

Comment:
Please be more specific in comment explaining what `nullptr` return value means

Reply:
Explained in the comment.